jest.config.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  collectCoverage: true,
+  collectCoverageFrom: ['./src/**/*.js'],
+  coverageDirectory: 'coverage',
+  coveragePathIgnorePatterns: ['/node_modules/'],
+  coverageThreshold: {
+    global: {
+      branches: 62,
+      functions: 84,
+      lines: 89,
+      statements: 89
+    }
+  },
+  moduleFileExtensions: ['js','json'],
+  rootDir: '.',
+  testEnvironment: 'node',
+  testMatch: ['**/*.spec.js'],
+  testResultsProcessor: 'jest-teamcity-reporter',
```

Comment:
This is not needed since it runs in Github actions.

Reply:
Removed